etc/profile-m-z/yt-dlp.profile

```diff
@@ -5,17 +5,65 @@ quiet
 # Persistent local customizations
 include yt-dlp.local
 # Persistent global definitions
-# added by included profile
-#include globals.local
+include globals.local
+
+# breaks when installed under ${HOME} via `pip install --user` (see #2833)
```

Comment:
Wouldn't it make more sense to keep `noexec ${HOME}` and add a new comment instead? Users who install it like this face apparmor issues anyway if they use that.

Reply:
I'd go for:

```
# If you installed via pip under ${HOME} add `ignore noexec ${HOME}`
# to your yt-dlp.local. AppArmor needs to allow it too, as per examples
# in /etc/apparmor.d/local/firejail-default.
noexec ${HOME}
```

Reply:
> isn't "noexec ${HOME}" already active from an inc?

I messed up here. That's correct.

> from what you said, i have no idea what to put in "/etc/apparmor.d/local/firejail-default"

https://github.com/netblue30/firejail/blob/1f5f84f9596ba7d26c2f2e35dad7728e12afaec9/etc/apparmor/firejail-local#L3-L10

Users with yt-dlp installed like that could go for the third example:
`owner @HOME/.local/bin/** ix,`

NOTE: I'll fix these examples, the comma at the end is needed in all three examples.

Reply:
This is a bit more involved than usual due to the combination of a local install and AppArmor. The thing is, if users (1) install yt-dlp via pip under /home/foo _and_ (2) are using AppArmor they can harden the Firejail sandbox by (3) adding those rules to /etc/apparmor.d/local/firejail-default. In that case they benefit from keeping the `apparmor` option.

Not sure if all that is conveyed in the current comment. To be clear, I'm not at all suggesting your comment is incorrect. I do appreciate your effort to make it as precise as possible for (Ubuntu) users. Not an easy job though to explain how to configure AA in a few lines. Maybe we can refer to distro documentation?

Reply:
> I rather not give them a "RTFM" comment.

No problem, I can dig that. Marking conversation as resolved.